### ISSUE TITLE
Support for `nbb`

### ISSUE THREAD

**Issue by dmg46664** [taoensso/timbre#358]
https://github.com/babashka/nbb
```
Welcome to nbb v1.0.136!
user=> (require '[taoensso.timbre :as timbre :refer-macros [info]])
"Error: Could not find namespace: goog.i18n.DateTimeFormat"
```

**Comment by ptaoussanis** [comment]
Hi there, I'm sorry but this issue isn't sufficiently clear for me to attempt a useful response.

Please don't expect library authors to read long linked documents without any context to explain what you're trying to achieve, or how the linked documentation might be relevant.

**Comment by dmg46664** [comment]
@ptaoussanis Apologies, thanks for responding.

I took the risk that you might somewhat familiar (and filed the feature request as a footnote while working around the problem writing my own logging library. Better to file it to start the conversation I thought, rather than not).

`nbb` is a javascript clojure interpreter by borkdude that is gaining popularity. I find it useful because I can amend aws lambda code and run it without re-compilation that `shadow-cljs` would require.

I've removed the `polylith` bit as it's only a potential workaround for my use case. As you say I'll need to expand on that for clarity.
